### PR TITLE
Add custom task failure notification doc

### LINKED ISSUE #1255: Custom task example: receive notification if task fails
Need to describe how to send a notification if a specific task fails.

@russorat has a working example that should be documented.

### PR DESCRIPTION
**Pull request by pierwill**
Closes #1255.

### CODE REVIEW

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Notify on failed tasks
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:2]: Alert on failed tasks

(think alert is preferable bc we have that term on top level)

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+seotitle: Notify on failed tasks.
+description: >
+  How to create a task to send a notification if a specific task fails.
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:5]: Create an alert email when a task fails.

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+menu:
+  influxdb_2_0:
+    name: Notify on failed tasks
+    parent: Common tasks
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:9]: parent: Monitor & alert

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+title: Notify on failed tasks
+seotitle: Notify on failed tasks.
+description: >
+  How to create a task to send a notification if a specific task fails.
+menu:
+  influxdb_2_0:
+    name: Notify on failed tasks
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:8]: Alert...

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+---
+title: Notify on failed tasks
+seotitle: Notify on failed tasks.
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:3]: Alert...

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+  How to create a task to send a notification if a specific task fails.
+menu:
+  influxdb_2_0:
+    name: Notify on failed tasks
+    parent: Common tasks
+weight: 201
+v2.0/tags: [tasks]
+---
+
+Use a task to monitor other tasks and receive alerts on failure.
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:14]: ...receive alerts when a task fails.

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+	|> monitor["check"](
+		data: check,
+		messageFn: messageFn,
+		ok: ok,
+		crit: crit,
+	)
+```
+
+## Add your task
+Once your task is ready, see [Create a task](/influxdb/v2.0/process-data/manage-tasks/create-task) for information about adding it to InfluxDB.
```
Comment (kelseiv) [inline content/influxdb/v2.0/process-data/common-tasks/task-failure.md:62]: Think the example script adds the task? So the script creates a "check" but doesn't send an email...

File: content/influxdb/v2.0/process-data/common-tasks/task-failure.md (hunk 6 above)
Comment (kelseiv) [reply content/influxdb/v2.0/process-data/common-tasks/task-failure.md:14]: We may want to clarify/verify whether this script creates an alert "check" only, and if it doesn't, we could add "next steps" and link to how to create an alert "email" or notification endpoints and rules.

**Comment by pierwill** [comment]
Closing this. Will address feedback in a new PR targeting `master`.

**Pull request closed without merging**